devtools/server/actors/replay/module.js

```diff
@@ -952,6 +953,17 @@ function Debugger_getPossibleBreakpoints({ sourceId, begin, end }) {
   }
 }
 
+function Target_getPossibleBreakpointsForMultipleSources({ sourceIds }) {
+  return { possibleBreakpoints:
+    sourceIds.map((sourceId => {
+      return {
+        sourceId,
+        ...Debugger_getPossibleBreakpoints({sourceId})
```

Comment:
This should use a helper function instead of invoking the other handler directly.